src/main/scala/wdl4s/examples/ex5.scala

```diff
@@ -22,10 +22,9 @@ object ex5 {
 
     val ns = NamespaceWithWorkflow.load(wdl)
 
-    Seq(ns.resolve("wf.a"), ns.resolve("wf.b")) foreach { scope =>
-      scope match {
-        case Some(c: Call) => println(s"Call '${c.fullyQualifiedName}' prerequisites: ${c.prerequisiteScopes}")
-      }
+    Seq(ns.resolve("wf.a"), ns.resolve("wf.b")) foreach {
+      case Some(c: Call) => println(s"Call '${c.fullyQualifiedName}' prerequisites: ${c.prerequisiteScopes}")
+      case _ => throw new RuntimeException("Should never get here")
```

Comment:
This is just moving code that was in Cromwell to wdl4s, you might want to look at that PR first.